### ISSUE TITLE
[4.0] Impossible to differentiate between a selectable option and a disabled option in a select html tag

### ISSUE THREAD

**Issue by obuisard** [joomla/joomla-cms#34496]
### Steps to reproduce the issue

Make sure the new workflow feature is not enabled on the site.
Go to Content -> Categories.
Edit a category. Go into the workflow tab.
Open the select box 'Workflows'.

### Expected result

You can differentiate between enabled and disabled options.

### Actual result

You cannot differentiate between enabled and disabled options. In this example (sorry, I could not find any better one), '-Workflows-' is disabled in the list.

![screen shot 2021-06-12 at 19 30 23](https://issues.joomla.org/uploads/1/bf22190ac77aaf6cebcdbe55a28c3d5f.png)

### System information (as much as possible)

Any Joomla 4 site.

### Additional comments

The core should take into account this kind of situation (even if rare), especially because third-party extensions may later use 'disabled' options in their fields.

**Comment by chmst** [comment]
In this case, the inherited basic workflow is selectable. This is the Joomla default (publish/ unpublish / trash / archive) and cannot be disabled. 
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34496">issues.joomla.org/tracker/joomla-cms/34496</a>.</sub>

**Comment by obuisard** [comment]
Thank you Christiane for your comment, I appreciate it.

This is actually _not_ my point here: I am talking about the fact that there is no CSS property in the admin template that allows users to differentiate between an option that is enabled and an option that is disabled. In the old school way, we would 'gray out' the disabled field, but this is not enough to make it accessible (not enough contrast).<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34496">issues.joomla.org/tracker/joomla-cms/34496</a>.</sub>

**Comment by brianteeman** [comment]
Please test #34616

**Comment by obuisard** [comment]
Closing this PR in favor of #38817
